kv-storage/src/rocks_db.rs

```diff
@@ -275,7 +281,31 @@ mod tests {
 
         assert!(get_test(&db, "key1", "val1").await);
         remove_test(&mut db, "key1").await;
-        assert!(!get_test(&db, "key1", "val1").await);
+    }
+
+    #[tokio::test]
+    async fn crud_integrate() {
+        let tmp_directory = init_db_ver1().await;
+        let mut db = RocksDB::open(tmp_directory.to_path_buf().to_str().unwrap())
+            .await
+            .unwrap();
+
+        assert!(get_test(&db, "key1", "val1").await);
+        assert!(get_test(&db, "key2", "val2").await);
+        assert!(get_test(&db, "key3", "val3").await);
+        assert!(get_test(&db, "key4", "val4").await);
+
+        remove_test(&mut db, "key1").await;
+        put_test(&mut db, "key1", "new_val1").await;
+        put_test(&mut db, "key3", "new_val3").await;
+        remove_test(&mut db, "key4").await;
+        put_test(&mut db, "key5", "val5").await;
+
+        assert!(get_test(&db, "key1", "new_val1").await);
+        assert!(get_test(&db, "key2", "val2").await);
+        assert!(get_test(&db, "key3", "new_val3").await);
+        assert!(!get_test(&db, "key4", "val4").await);
+        assert!(get_test(&db, "key5", "val5").await);
```

Comment:
All these codes are redundant since we check the same way in `put_test` and `remove_test`.

Reply:
fix